application/F3DStarter.cxx

```diff
@@ -1636,4 +1639,13 @@ void F3DStarter::AddCommands()
         this->LoadFileGroup(index);
       }
     });
+    interactor.addCommand("open_file", 
+      [this](const std::vector<std::string>& args)
```

Comment:
```suggestion
      [this](const std::vector<std::string>&)
```

Reply:
This should be fixed now. I've mostly been coding in C so I didn't know you could do this in a lambda.

Reply:
Feel free to resolve discussion you adress :)